packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -30,6 +30,59 @@ import 'tooltip_theme.dart';
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=EeEfD5fI-5Q}
 ///
+/// ## Example
+///
+/// {@tool dartpad --template=stateless_widget_material}
+///
+/// This example show a basic [Tooltip] which has a [Text] as child.
+/// [message] contains your label to be shown on long pressed.
+///
+/// ```dart
+/// Widget build(BuildContext context) {
+///   return Center(
+///     child: Tooltip(
+///       message: "Welcome Greetings",
+///       child: Text("Hello Flutter"),
+///     ),
+///   );
+/// }
+/// ```
+/// {@end-tool}
+///
+/// {@tool dartpad --template=stateless_widget_material}
+///
+/// This example covers most of the attributes available in Tooltip.
+/// `decoration` has been used to give a gradient and borderRadius to Tooltip.
+/// `height` has been used to set a specific size of the Tooltip.
```

Comment:
There's no "width" parameter, so let's be a little more specific here
```suggestion
/// `height` has been used to set a specific height of the Tooltip.
```